Makefile

```diff
@@ -8,8 +8,8 @@ help:               ## - Show this help.
 generate:           ## - Generate test cases (do this once before running test)
 	poetry run python -m tests.generate
 
-test:               ## - Run tests
-	poetry run pytest --cov betterproto
+test:               ## - Run tests, ingoring collection errors (ex from missing imports)
+	poetry run pytest --cov betterproto --continue-on-collection-errors
```

Comment:
This is obviously not needed for this PR, but I found it very useful, will remove before final review.